### PR TITLE
odigrala 1.potez u obe igre

### PR DESCRIPTION
**Pull request by mioc333**

### CODE REVIEW

File: game1.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Antonia ima 0 pobjeda i 0 poraza!
 Andrea ima 0 pobjeda i 0 poraza!
 
-. . .
+. O .
```
Comment (amales24) [inline game1.txt:4]: you're gonna lose

File: game2.txt [hunk 2]
```diff
@@ -2,7 +2,7 @@ Antonia ima 0 pobjeda i 0 poraza!
 Andrea ima 0 pobjeda i 0 poraza!
 
 . . .
-. . .
+. O .
```
Comment (amales24) [inline game2.txt:5]: ok fair play

**Review by amales24 — APPROVED** [review]
game on!

File: game2.txt (hunk 2 above)
Comment (mioc333) [reply game2.txt:5]: tnx bro

**Pull request merged**